### ISSUE TITLE
Assigning actor.image removes angle transform

### ISSUE THREAD

**Issue by lordmauve** [lordmauve/pgzero#124]
When a new image is assigned to an actor that already has `angle` set to something other than 0, the angle value is not respected (the image appears untransformed).

**Comment by charlesej** [comment]
I think this issue has already been resolved by PR #130.

Using the code below, I was able to reproduce this issue in pgzero 1.2. However, in the current master branch the `.angle` value is respected when the image is drawn.

```
alien = Actor('alien', topleft=(0,0))
alien.angle = 45
alien.image = 'alien2'

def draw():
    alien.draw()
```

System details:
- os: windows 10
- python: 3.5.0
- pygame: 1.9.4
- pgzero: 1.2 and master branch at 8dc6800960d58f886118954f6756c87e45c91302
- numpy: 1.15.4

**Comment by lordmauve** [comment]
Great, thanks for investigating!
